build/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+export AWS_DEFAULT_REGION=us-east-2
+export AWS_ACCESS_KEY_ID=test
+export AWS_SECRET_ACCESS_KEY=test
+export ENDPOINT=http://localstack:4566
+
+timeout=120 #seconds
+while [ "$(curl -s $ENDPOINT/health | jq '.services.s3')" != "\"running\"" ]; do
```

Comment:
we can probably get away with the jq dependency by using aws cli to communicate with localstack endpoints and sleep on error.